README.md

```diff
@@ -119,6 +121,25 @@ This is an example of the value of `Specberus.meta` after the execution of `Spec
 }
 ```
 
+## REST API
+
+Similar to the [Node.js API](#js-api), Specberus exposes a REST API too.
+
+The endpoint is `<host>/api/`.
+
+There are three `GET` methods available:
+
+* `version`: returns the version string, eg `1.5.3`.
+* `metadata`: extract all known metadata from document; [returns JSON object with inferred properties](#extractmetadataoptions).
+* `validate`: check the document ([syntax](#validateoptions)); fails and returns an array of errors, or succeeds and returns the profile.
+
+Examples:
+
+* `<host>/api/version`
+* `<host>/api/metadata?url=http://example.com/doc.html`
+* `<host>/api/validate?file=/home/me/docs/spec.html`
```

Comment:
`file` and `document` are definitely useful but so far, we know, it's not working properly and these are new features compared to the current pubrules, so :+1: to disabling these options from the REST API for now.